src/local-run.ts

```diff
@@ -2,9 +2,17 @@ import { createManifest, readAll } from "./deps.ts";
 import { ParsePayload } from "./parse-payload.ts";
 import { DispatchPayload } from "./dispatch-payload.ts";
 
-export const runLocally = async function () {
+/**
+ * @description Runs an application function locally by dispatching a payload to it after loading it.
+ */
+export const runLocally = async function (
+  create: typeof createManifest,
+  parse: typeof ParsePayload,
+  readStdin: typeof readAll,
+  dispatch: typeof DispatchPayload,
```

Comment:
This is the 'dependency injection' change. When run from the command line, as the CLI does by invoking the relevant hook, the last lines in this module will run, which inject the default `createManifest`, `ParsePayload`, `readAll` and `DispatchPayload` functional dependencies. The only other use of this method is in tests, which let us inject mocks or other utilities for easy testing.